app/assets/stylesheets/custom.css.scss

```diff
@@ -133,35 +139,40 @@ body {
   padding: 0;
   font-size: 4.5em;
   font-weight: 700;
-  color: gray;
+  color: $muted;
 }
 
 /* Custom Button Styles */
 
-.btn-dark {
-  border-radius: 0;
-  color: $white;
-  background-color: rgba(0,0,0,0.4);
+.btn-xlarge {
+  padding: 18px 28px;
+  font-size: 22px;
+  line-height: normal;
+  -webkit-border-radius: 8px;
+  -moz-border-radius: 8px;
+  border-radius: 8px;
 }
 
-.btn-dark:hover,
-.btn-dark:focus,
-.btn-dark:active {
-  color: $white;
-  background-color: rgba(0,0,0,0.7);
+.btn-ghost {
+  color: $primaryColor !important;
```

Comment:
!important should not be used<br>Name of variable `primaryColor` should be written in all lowercase letters with hyphens instead of underscores<br>Properties should be ordered background-color, border, border-radius, color, font-weight